### PR TITLE
Ca configmap cleanup

### PR DESCRIPTION
**Pull request by MaciekPytel**

### CODE REVIEW

File: cluster-autoscaler/run.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# cluster-autoscaler binary.
+
+if [ -z "$LOG_OUTPUT" ]; then
+  LOG_OUTPUT="/var/log/cluster_autoscaler.log"
+fi
+
+./cluster-autoscaler $@ 1>>$LOG_OUTPUT 2>&1 &
+pid="$!"
+trap "kill -15 $pid" 15
+while kill -0 $pid > /dev/null 2>&1; do
```
Comment (mwielgus) [inline cluster-autoscaler/run.sh]: Add a comment about -0.

Comment (mwielgus) [inline cluster-autoscaler/run.sh]: And why do we have a loop here - it is not obvious.

Comment (MaciekPytel) [inline cluster-autoscaler/run.sh]: done

**Pull request merged**